tests/cpp/test_mma.cpp

```diff
@@ -441,6 +441,254 @@ TEST_P(HopperRS, SingleTile) {
   EXPECT_TRUE(at::allclose(cg_outputs[0], tref, 1e-5, 1e-5));
 }
 
+void scheduleTMALoadWhereOuterDimIsSplit(
```

Comment:
Should we create a member function of TensorView that contains common part of this and `scheduleTMALoadOuterDimNotSplit`? Like `TensorView::applySwizzle(MmaInputSmemSwizzle)`?

Reply:
That's a good ideas - I'll send a new PR for that.